modules/microshift-certificate-lifetime.adoc

```diff
@@ -11,17 +11,17 @@
 . Short-lived certificates having certificate validity of one year.
 . Long-lived certificates having certificate validity of 10 years.
 
-Most server or leaf certificates are short-term.
+Most server or leaf certificates are short-lived.
 
 An example of a long-lived certificate is the client certificate for `system:admin user` authentication, or the certificate of the signer of the `kube-apiserver` external serving certificate.
 
 [id="microshift-certificate-rotation_{context}"]
 == Certificate rotation
 Certificates that are expired or close to their expiration dates need to be rotated to ensure continued {microshift-short} operation. When {microshift-short} restarts for any reason, certificates that are close to expiring are rotated. A certificate that is set to expire imminently, or has expired, can cause an automatic {microshift-short} restart to perform a rotation.
 
-[NOTE]
+[IMPORTANT]
 ====
-If the rotated certificate is a Certificate Authority, all of the certificates it signed rotate.
+If the rotated certificate is a {microshift-short} Certificate Authority, all of the certificates it signed rotate. If you created any Custom Certificate Authorities, they must be rotated manually.
```

Comment:
```suggestion
If the rotated certificate is a {microshift-short} certificate authority (CA) then all of the signed certificates rotate. If you created any custom CAs, ensure the CAs manually rotate.
```

Certificate authority needs to be highlighted? https://redhat-documentation.github.io/supplementary-style-guide/#certificate-authority